addons/point_of_sale/static/src/app/screens/receipt_screen/receipt/order_receipt.js

```diff
@@ -18,4 +18,10 @@ export class OrderReceipt extends Component {
     omit(...args) {
         return omit(...args);
     }
+    doAllOrderlinesHaveTaxes() {
+        return this.props.data.orderlines.some((line) => line.taxGroupLabels);
```

Comment:
the implementation doesn't reflect the name of the method. if you want "all", shouldn't you be using the `Array.every` method?

Reply:
Changed the name to `doesAnyOrderlineHaveTaxLabel`. Basically we want to show the tax label column if at least one order line has a tax label